compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -109,7 +109,7 @@ namespace TR { class RegisterDependencyConditions; }
 
 TR_X86ProcessorInfo OMR::X86::CodeGenerator::_targetProcessorInfo;
 
-void TR_X86ProcessorInfo::initialize()
+void TR_X86ProcessorInfo::initialize(TR::CodeGenerator *cg)
```

Comment:
This is tricky; even though `initialize` should only be called once, it is actually before every, so it's fine that we'd be querying the the comp-local target env. However, eventually this should be moved out into the place where the target is defined (either by the user or some preset config).

Reply:
Can we open an issue to track this so we don't forget?